docs/api-docs/storefront/graphql/graphql-api-overview.md

```diff
@@ -313,6 +215,97 @@ In addition to using `fetch()`, there's a other ways to query the API:
 </div>
 </div>
 
+<a id="accessing-the-graphql-playground" class="devdocsAnchor"></a>
+
+## Accessing the GraphQL Playground
+
+To access the GraphQL Storefront API Playground<sup>1</sup> and documentation:
+
+1. Login to a BigCommerce store enrolled in the beta
+2. Navigate to **Advanced Settings** > **Storefront API Playground**<sup>2</sup>
+
+The GraphQL Storefront API Playground will be opened:
+
+![GraphQL Storefront API Playground](https://raw.githubusercontent.com/bigcommerce/dev-docs/master/assets/images/graphql-storefront-api-playground.png "GraphQL Storefront API Playground")
+
+<div class="HubBlock--callout">
+<div class="CalloutBlock--info">
+<div class="HubBlock-content">
+    
+<!-- theme: info -->
+
+### Note

```

Comment:
The two notes listed here should be bulleted instead of numbered. Numbering implies both instructions to be followed and specific order.